File: Server/src/main/java/com/augmentedcooking/Enums/Response/ResponseStatus.java
package com.augmentedcooking.Enums.Response;

import com.fasterxml.jackson.annotation.JsonValue;

public enum ResponseStatus {
    SUCCESS("success"),
    ERROR("error");

    private final String status;

    ResponseStatus(String status) {
        this.status = status;
    }

    @JsonValue
    public String status() {
        return status;
    }
}

File: Clients/desktop/src/Enums/ResponseStatus.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using System.Runtime.Serialization;

namespace AugmentedCooking.src.Enums {
    [JsonConverter(typeof(StringEnumConverter))]
    public enum ResponseStatus {
        [EnumMember(Value = "success")]
        Success,

        [EnumMember(Value = "error")]
        Error
    }
}
